test_vm/tests/evm_test.rs

```diff
@@ -231,34 +239,46 @@ fn test_evm_create() {
 }
 
 #[test]
-fn test_evm_eth_create_external() {
+fn evm_eth_create_external() {
     let store = MemoryBlockstore::new();
     let v = TestVM::<MemoryBlockstore>::new_with_singletons(&store);
 
+    evm_eth_create_external_test(&v, &v);
+}
+
+// TODO: drop usage of the concrete TestVM
```

Comment:
Please describe what's blocking doing it now, and link to an issue to resolve that.